src/Nox.Types.EntityFramework/Configurations/NoxDatabaseConfigurator.cs

```diff
@@ -313,5 +325,32 @@ private static void ConfigureConstraintsWithAuditProperties(IEntityBuilder build
                 }
             }
         }
+
+        protected virtual void ConfigureLocalizedAttributes(
+            NoxSolutionCodeGeneratorState codeGeneratorState,
+            IEntityBuilder builder,
+            Entity entity)
+        {
+            var attributes = entity.GetAttributesToLocalize().ToList();
+            attributes.Add(new NoxSimpleTypeDefinition
+            {
+                Name = "CultureCode",
```

Comment:
maybe the hardcoded cultureCode we can promote to NoxCodeGenConventions and also update our templates to use it?

Reply:
Done